src/ripple/app/main/LoadManager.cpp

```diff
@@ -134,11 +134,15 @@ void LoadManager::run ()
                         << timeSpentDeadlocked.count() << " seconds.";
                 }
 
-                // If we go over 500 seconds spent deadlocked, it means that
+                // If we go over 90 seconds spent deadlocked, it means that
                 // the deadlock resolution code has failed, which qualifies
                 // as undefined behavior.
                 //
-                assert (timeSpentDeadlocked < 500s);
+                constexpr auto deadlockTimeLimit = 90s;
+                assert (timeSpentDeadlocked < deadlockTimeLimits);
```

Comment:
typo on this line (trailing `s`)

Reply:
How did this compile on MSVC (see CI table, above)? 😕

Reply:
On MSVC the entire assert expression is removed in release builds 🙃, Which is coincidentally what I was compiling before... but I had to test on Linux and I didn't want to wait the full 90s so I actually changed this to 15s before compiling. I probably fixed the typo to get it to build while testing and forgot about it (since MSVC was building fine locally).